--- /termite.cc	2016-10-02 18:12:24.000000000 +0100
+++ /termite.cc	2016-12-29 00:51:58.496272107 +0000
@@ -914,6 +914,19 @@
         }
         return TRUE;
     }
+    if (modifiers == GDK_SUPER_MASK) {
+        switch (gdk_keyval_to_lower(event->keyval)) {
+            case GDK_KEY_c:
+                vte_terminal_copy_clipboard(vte);
+                return TRUE;
+            case GDK_KEY_v:
+                vte_terminal_paste_clipboard(vte);
+                return TRUE;
+            default:
+                if (modify_key_feed(event, info, modify_table))
+                    return TRUE;
+        }
+    }
     if (modifiers == (GDK_CONTROL_MASK|GDK_SHIFT_MASK)) {
         switch (gdk_keyval_to_lower(event->keyval)) {
             case GDK_KEY_t:
